src/player.jsx

```diff
@@ -1236,168 +1242,220 @@ export class Player extends React.Component {
-                            </div>
-                            <div className="panel-body">
-                                <table className="form-table-ct">
-                                    <tbody>
-                                        <tr>
-                                            <td>{_("ID")}</td>
-                                            <td>{r.id}</td>
-                                        </tr>
-                                        <tr>
-                                            <td>{_("Hostname")}</td>
-                                            <td>{r.hostname}</td>
-                                        </tr>
-                                        <tr>
-                                            <td>{_("Boot ID")}</td>
-                                            <td>{r.boot_id}</td>
-                                        </tr>
-                                        <tr>
-                                            <td>{_("Session ID")}</td>
-                                            <td>{r.session_id}</td>
-                                        </tr>
-                                        <tr>
-                                            <td>{_("PID")}</td>
-                                            <td>{r.pid}</td>
-                                        </tr>
-                                        <tr>
-                                            <td>{_("Start")}</td>
-                                            <td>{formatDateTime(r.start)}</td>
-                                        </tr>
-                                        <tr>
-                                            <td>{_("End")}</td>
-                                            <td>{formatDateTime(r.end)}</td>
-                                        </tr>
-                                        <tr>
-                                            <td>{_("Duration")}</td>
-                                            <td>{formatDuration(r.end - r.start)}</td>
-                                        </tr>
-                                        <tr>
-                                            <td>{_("User")}</td>
-                                            <td>{r.user}</td>
-                                        </tr>
-                                    </tbody>
-                                </table>
-                            </div>
+                    <div className="panel-body">
+                        <div
+                            className={(this.state.drag_pan ? "dragnpan" : "")}
+                            style={scrollwrap}
+                            ref="scrollwrap">
+                            <div
+                                ref="term"
+                                className="console-ct"
+                                key={this.state.term}
+                                style={style} />
                         </div>
                     </div>
+                    <Progress
```

Comment:
Can we create a **progressBar** constant which is defined above in render(), similar to `const recordingInfo` and `const panel`, or maybe this can be included in `panel`.

Then we are looking in the same place for the components to render, and at some point we may want to extract these variables like panel into their own file/module.